Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.html

```diff
@@ -6,423 +6,546 @@
+	      <ul class="pagination">
+	         <li class="page-item" th:classappend="${page == 1} ? disabled : '' ">
+	            <span class="page-link" th:data-sortdir="${sortDir}" th:data-sortColumn="${sortColumn}"
+	               th:data-searchword="${conceptSearchBean.word}" th:data-searchpos="${conceptSearchBean.pos}" th:data-page="${page - 1}" th:data-numRecords="${numRecords}" id="paginationTag">Previous</span>
+	         </li>
+	         <li class="page-item" th:classappend="${page == 1} ? disabled : '' ">
+	            <span class="page-link" th:data-sortdir="${sortDir}" th:data-sortColumn="${sortColumn}"
+	               th:data-searchword="${conceptSearchBean.word}" th:data-searchpos="${conceptSearchBean.pos}" th:data-page="1" th:data-numRecords="${numRecords}" id="paginationTag">First</span>
+	         </li>
+	         <li th:each="i: ${#numbers.sequence(1, count)}" th:class="page-item" th:classappend="${page == i} ? active: '' ">
+	            <span class="page-link" th:data-sortdir="${sortDir}" th:data-sortColumn="${sortColumn}" 
+	               th:data-searchword="${conceptSearchBean.word}" th:data-searchpos="${conceptSearchBean.pos}" th:data-page="${i}" th:data-numRecords="${numRecords}" id="paginationTag" th:text="${i}">
+	            <span class="sr-only">(current)</span></span>
+	         </li>
+	         <li class="page-item" th:classappend="${page == count} ? disabled : '' ">
+	            <span class="page-link" th:data-sortdir="${sortDir}" th:data-sortColumn="${sortColumn}" 
+	               th:data-searchword="${conceptSearchBean.word}" th:data-searchpos="${conceptSearchBean.pos}" th:data-page="${count}" th:data-numRecords="${numRecords}" id="paginationTag">Last</span>
+	         </li>
+	         <li class="page-item" th:classappend="${page == count} ? disabled : '' ">
+	            <span class="page-link" th:data-sortdir="${sortDir}" th:data-sortColumn="${sortColumn}" 
+	               th:data-searchword="${conceptSearchBean.word}" th:data-searchpos="${conceptSearchBean.pos}" th:data-page="${page + 1}" th:data-numRecords="${numRecords}" id="paginationTag">Next</span>
+	         </li>
+	      </ul>
+	   </div>
+	   <input type='hidden' id='deleteUrl'/>
+	   <input type="hidden" name="reviewId" id="reviewId" value=""/>
+	   <input type="hidden" name="conceptId" id="conceptId" value=""/>
+	   
+	   <div th:insert="layouts/modals/conceptdetails :: conceptdetails"></div>
+	   <div th:insert="layouts/modals/requestreview :: addrequest"></div>
+	   <div th:insert="layouts/modals/resolvereview :: resolverequest"></div>
+	   <div th:insert="layouts/modals/reopenreview :: reopenrequest"></div>
+	   <div th:insert="layouts/modals/invalidrequest :: invalidRequest"></div>
+	   <div sec:authorize="!isAuthenticated()" th:insert="layouts/modals/signin :: signin"></div>
+	   <div th:insert="layouts/modals/deleteconcept :: deleteConcept"></div>
+	</body>
```

Comment:
what has actually been changed in this file? Please don't change parts that don't need to be changed.

Reply:
I'm sorry, yeah. I have removed a couple of AJAX functions and put them in a common js file.

Reply:
please revert formatting changes and reread PR guidelines.